### DISCUSSION TITLE
节点选中框层级如何修改？

### DISCUSSION THREAD

**Discussion by dzb0421** [antvis/X6#3804] [Q&A]
![企业微信截图_16902828255754](https://github.com/antvis/X6/assets/42457225/47291bc3-98a8-44e3-bb6d-fb68d7a495b8)
当在画布中选中节点会出现选中框，如果拖拽节点到画布外，选中框依然可见，如何修改呢？

**Comment by NewByVector** [comment]
可以讲最外侧容器的 `overflow` 设置为 `hidden`。

**Reply by dzb0421** [reply]
谢谢你，解决了
